browser/js/common/directives/navbar/navbar.js

```diff
@@ -18,8 +18,8 @@ app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state,
 
             var setUser = function () {
                 return UserFactory.getUser()
-                .then(function (foundUser) {
-                    scope.user = foundUser;
+                .then(() => {
+                    scope.user = UserFactory.getCachedUser();
```

Comment:
Why do you have to get user from user factory and then also get the cached user?  Shouldn't the user already have been returned from .getUser()